merkletree/pad.go

```diff
@@ -10,7 +10,13 @@ import (
 )
 
 var (
+	// ErrorSTRNotFound indicates that the STR of the searching epoch
+	// has been removed from the memory, because the snapshot array has exceeded
+	// its maximum length.
```

Comment:
This explanation may not make that much sense to a developer since the snapshot array isn't directly exposed (and there's currently no specification for our PAD design). Maybe say something like "the STR has been evicted from memory, because the maximum number of cached PAD snapshots has been exceeded" is a bit clearer?